scripts/collision_detection_example.py

```diff
@@ -73,21 +91,38 @@ def main():
         [(link7, ground), (link7, cube1), (link7, cube2), (link7, cube3)],
     )
 
+    robot_id = bodies["robot"]
+
     while True:
         # compute shortest distances for a random configuration
-        q = np.pi * (np.random.random(7) - 0.5)
+        q = np.array([
```

Comment:
Can we make this a function, say `read_robot_state`, which populates `q` with a loop over the debug params?

Reply:
I have moved reading debug params to `read_robot_state` but I am still manually setting the state array. The reason is that reading them over a loop was making the robot state change in GUI jittery